src/components/nav.js

```diff
@@ -84,13 +66,11 @@ const Button = styled.a`
 
 const Nav = ({ page, pages }) => {
   const [atTop, setAtTop] = useState(true)
-  const [hasScrolled, setHasScrolled] = useState(false)
```

Comment:
What the heck? Was this even doing anything? I definitely wrote this on that super late night in Dav lol

Reply:
yeah it was useful for hiding the nav on the home page, but we don't do that anymore